test/release/index.js

```diff
@@ -111,36 +59,16 @@ const pages = [
         "key": "locate-user",
         "title": "Locate the user"
     },
-    {
-        "key": "extrusion-query",
-        "url": "./extrusion-query.html",
-        "title": "Fill extrusion querying with terrain"
-    },
```

Comment:
I'm not sure if fill-extrusion querying with terrain is tested elsewhere and truly redundant, but I also wonder if the testing coverage of query tests to be sufficient for this one. It's probably ok to remove.

Reply:
fill-extrusion querying with terrain is included in 3d-playground and fog

Reply:
>fill-extrusion querying with terrain is included in 3d-playground and fog

I only see terrain elevation querying in those pages?

Reply:
This is my mistake. just setting fill extrusion, not querying in those pages. We can add this back in and split up 3d-playground in favor of more straight forward testing pages for terrain and sky.

Reply:
No worries, I checked and it's covered by some query test so a good call to remove.